basiclti/basiclti-common/src/java/org/sakaiproject/lti13/LineItemUtil.java

```diff
@@ -106,16 +106,25 @@ public static String constructExternalId(Long tool_id, Map<String, Object> conte
 		return retval;
 	}
 
-	public static Assignment createLineItem(Site site, Long tool_id, Map<String, Object> content, LineItem lineItem) {
+	public static Assignment createLineItem(Site site, Long tool_id, Map<String, Object> content, SakaiLineItem lineItem) {
+		String context_id = site.getId();
+		return createLineItem(context_id, tool_id, content, lineItem);
+	}
+
+	public static Assignment createLineItem(String context_id, Long tool_id, Map<String, Object> content, SakaiLineItem lineItem) {
 		// Look up the assignment so we can find the max points
 		GradebookService g = (GradebookService) ComponentManager
 				.get("org.sakaiproject.service.gradebook.GradebookService");
 
-		String context_id = site.getId();
 		if (lineItem.scoreMaximum == null) {
 			lineItem.scoreMaximum = 100.0;
 		}
 
+System.out.println("createLineItem="+lineItem.toString());
+System.out.println("SakaiLineItem.label="+lineItem.label);
+System.out.println("SakaiLineItem.scoreMaximum="+lineItem.scoreMaximum);
+System.out.println("SakaiLineItem.includeInComputation="+lineItem.includeInComputation);
+System.out.println("SakaiLineItem.releaseToStudent="+lineItem.releaseToStudent);
```

Comment:
system out

Reply:
Dang - I looked for those and missed them - thanks.